### PR TITLE
Disconnected envt support for Pipelines

### PR DESCRIPTION
**Pull request by Preeticp**
This PR adds a note for Disconnected environment support
This should be merged post Pipelines TP release 
This should go in the 4.6 and later versions.

### CODE REVIEW

**Review by ppitonak — APPROVED** [review]

**Comment by ppitonak** [comment]
Do I understand it correctly that this change will only go to docs for 4.6 and newer? I'm not sure if we shouldn't mention Pipelines version (1.2) explicitly so that we don't cause confusion, especially for the time before promoting 1.2 to `ocp-4.6` channel.

**Comment by Preeticp** [comment]
@ppitonak, you are right, this goes in 4.6 and later, but we can ensure that this gets merged only after our TP 1.2 release. Will that still be a concern?

**Comment by ppitonak** [comment]
I think that after TP 1.2 it would be clear enough. Now I realized that in disconnected, the community cluster tasks won't be installed, do we want to mention that?

**Comment by Preeticp** [comment]
As discussed we do not document community cluster tasks in our docs, so going ahead and asking the OCP team to review. Thank you.

File: pipelines/installing-pipelines.adoc [hunk 1]
```diff
@@ -6,6 +6,13 @@ include::modules/pipelines-document-attributes.adoc[]
 
 toc::[]
 
+This guide walks cluster administrators through installing the {pipelines-title} Operator to an {product-title} cluster.
```
Comment (bobfuru) [inline pipelines/installing-pipelines.adoc:9]: `through installing` -> `through the process of installing` ?

**Review by bobfuru — COMMENTED** [review]
One suggestion and if you could also squash commits and then let the team know when this is OK to merge, please. Otherwise, LGTM!

File: pipelines/installing-pipelines.adoc (hunk 1 above)
Comment (Preeticp) [reply pipelines/installing-pipelines.adoc]: fixed

**Comment by Preeticp** [comment]
Thank you, Bob, all done. This just needs a merge on the day of TP

**Comment by bobfuru** [comment]
> Thank you, Bob, all done. This just needs a merge on the day of TP

Sounds good, @Preeticp. Please ping @peer-review-squad in Slack to have one of the squad members merge this when you are ready on the day of TP.

**Pull request merged**